app/views/offline/offline.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="<%= SiteSetting.default_locale %>" xml:lang="<%= SiteSetting.default_locale %>">
+
+<head>
+  <meta http-equiv="Content-type" name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=no, width=device-width">
+  <title></title>
+</head>
+
+<div style="padding:100px 40px; text-align:center; font-family: Helvetica, Arial, sans-serif;">
+  <h1 style="font-size: 36px; color: #333;"><%= t 'offline.title' %></h1>
+  <p style="margin: 40px; font-size: 18px; color: #555;">
+    <%= t 'offline.offline_page_message' %>
+  </p>
+</div>
+</body>
```

Comment:
Seems like the opening `<body>` tag is missing?

Reply:
Yes, fixed it.